composer.json

```diff
@@ -3,7 +3,7 @@
     "type": "library",
     "description": "Database Schema migrations using Doctrine DBAL",
     "keywords": ["migrations", "database"],
-    "homepage": "http://www.doctrine-project.org",
+    "homepage": "https://www.doctrine-project.org",
```

Comment:
Should we link to the project page for migrations instead of the root?

Reply:
Not sure about that, haven't seen such pattern anywhere. :/

Reply:
How about adding a `support` section, then?

For reference, see https://getcomposer.org/doc/04-schema.md#support.

Reply:
Well, once we have a proper page for the product, it would make sense to link to it as the homepage.

Reply:
It should link here https://www.doctrine-project.org/projects/migrations.html